dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteBrowserResource.java

```diff
@@ -208,5 +208,48 @@ public final Response switchSite(
 
         return response;
     } // sites.
+    
+    /**
+	 * Returns the list of Sites that the currently logged-in user has access
+	 * to. In the front-end, this list is displayed in the Site Selector
+	 * component. Its contents will also be refreshed when performing the "Login
+	 * As".
+	 * <p>
+	 * The site that will be selected in the UI component will be retrieved from
+	 * the HTTP session. If such a site does not exist in the list of sites, the
+	 * first site in it will be selected.
+	 * 
+	 * @param req
+	 *            - The {@link HttpServletRequest} object.
+	 * @return The {@link Response} containing the list of Sites.
+	 */
+    @GET
+    @Path ("/paginatedSites/filter/{filter}/archived/{archived}/page/{page}/limit/{limit}")
```

Comment:
Neither a filter, nor a page number, nor a limit should be part of the resource path. They should be query params. Also why creating a new path 'paginatedSites' ? Why not using the existing /site and have optional query params? like /site?page=2&limit=50. See https://developer.github.com/v3/#pagination as reference.